src/blocksmith/plugins/FloatingTextFormatToolbarPlugin.js

```diff
@@ -192,16 +190,15 @@ function FloatingTextFormatToolbar({
         >
           <Icon size="sd" imgSrc="underline" />
         </button>
-        {!isLink ? (
-          <button
-            onClick={handleInsertLink}
-            className="toolbar-item"
-            aria-label="Insert link"
-            type="button"
-          >
-            <Icon imgSrc="link" />
-          </button>
-        ) : (
+        <button
+          onClick={handleInsertLink}
```

Comment:
@joshlevy nit: should probably call this `handleOpenLinkModal` or something because it won't always be inserting anymore